otx/algorithms/classification/adapters/deep_object_reid/configs/configuration.yaml

```diff
@@ -0,0 +1,161 @@
+description: Configuration for an image classification task
```

Comment:
Ideally adapters should not have yaml configurations. It could contain some modules that "adapts" `dor` or `mmcls` configuration to ote format, but any yaml configuration should be on the main `config` directory of the task

Reply:
In my opinion, backend-specific configurations could be located in adapters.
BTW, we'll remove the deep-object-reid adapters as soon as possible so you don't need to care about this too much :)

Reply:
I couldn't help but leave yaml files. Because DOR is not just module, running nncf task for otx classification. nncf related codes are in dor. Its configs are very different from mmcls. If not having yaml files for dor, nncf stops incurring related errors. We plan to run nncf by mmcls soon just after this refactoring task. DOR will be completely removed then.

Reply:
Thanks for the clarification @goodsong81 and @supersoob. Much clearer now!